content/updates/2019-07-12-VPN-backhaul-drupal-8-and-new-team-members.md

```diff
@@ -0,0 +1,89 @@
+
+### Now available: Connect your cloud.gov apps to your private network
+
+Last October, we announced our team was working on a way for cloud.gov applications to interact securely with your applications on other networks through a VPN, such as systems in an agency data center. This VPN connection can make it easier to run cloud-native applications that need to communicate with legacy systems.
+
+We are so excited to announce [our VPN backhaul](https://cloud.gov/docs/apps/private-egress/) is officially available, and we can configure it for customers who need it! [Contact us if you’d like this to use this feature](https://cloud.gov/docs/apps/private-egress/#how-to-get-a-dedicated-pool-of-cloud-gov-hosts-connected-to-your-organization), and we’ll work with the appropriate teams in your agency to get you set up.
+
+### 2019 FedRAMP annual assessment completed, cloud.gov granted re-authorization
+
+Every year, to maintain [our FedRAMP Authorization](https://cloud.gov/overview/security/fedramp-tracker/), cloud.gov goes through a robust security compliance audit to ensure our security measures are up to date with current federal standards. This enables our customer systems to inherit a significant amount of compliance from the cloud.gov platform, supporting faster ATOs and less work for their teams.
+
+Our team has successfully completed this year’s re-authorization. Current and prospective customers can view our audit materials in the FedRAMP document repository on MAX.gov by [requesting access from FedRAMP](https://cloud.gov/overview/security/fedramp-tracker/#start-the-ato-process).
+
+## New additions
+
+
+### Try an experimental feature for no-downtime deployments
+
+It’s often a challenge to build a process for deploying new versions of your applications with zero downtime, without needing to double your memory usage. The [latest version of the Cloud Foundry command-line interface (CLI)](https://github.com/cloudfoundry/cli/releases) includes new beta commands to push apps using [rolling deployment](https://docs.cloudfoundry.org/devguide/deploy-apps/rolling-deploy.html). This makes it much easier to implement [blue-green deployment patterns](https://docs.cloudfoundry.org/devguide/deploy-apps/blue-green.html) without doubling your memory quota. (For example, this can replace the unmaintained [autopilot plugin](https://github.com/contraband/autopilot).) Install [the CF CLI update](https://github.com/cloudfoundry/cli/releases), try it out, and let us know if you have any questions!
+
+### Functions support for MySQL databases
+
+If you need to set up a MySQL schema that includes functions, procedures, and triggers, you can now create new databases that have that feature enabled. [To do this, see the “Options” documentation for RDS databases.](https://cloud.gov/docs/services/relational-database/#options) This allows you to build a wider range of applications on cloud.gov.
+
+### Improved instructions for running Drupal 8 for managing content
```

Comment:
I'm on the fence about this. I don't want to be supporting custom applications, and by discussing it in the release notes gives a lot of smells that we support it. I think it might be good to reframe this as a only a sample.

Reply:
Removed the "contact us" and just provide samples